CHANGELOG.md

```diff
@@ -78,6 +79,9 @@ Ref: https://keepachangelog.com/en/1.0.0/
   * `MsgCreateValidator.Pubkey` type changed from `string` to `codectypes.Any`.
 * (client) [\#8926](https://github.com/cosmos/cosmos-sdk/pull/8926) `client/tx.PrepareFactory` has been converted to a private function, as it's only used internally.
 * (auth/tx) [\#8926](https://github.com/cosmos/cosmos-sdk/pull/8926) The `ProtoTxProvider` interface used as a workaround for transaction simulation has been removed.
+* [#9088](https://github.com/cosmos/cosmos-sdk/pull/9088) Unify naming of the ADR-28 related functions: Renames: `address.NewCompose` -> `address.Compose`, `address.NewCompose`
```

Comment:
I'm never sure about these breaking change entries. So it's obvisouly a breaking change compared to master, but it's NOT a breaking change compared to 0.42. Adding this line means more work of the unlucky hero who needs to clean up changelogs for the 043 release. I would propose just to remove this line. cc @alessio @clevinson

Reply:
Agree, sounds reasonable. Let's check if Alessio or Cory has other opinion, if not I will merge changelog entries.

Reply:
So what's the decision about Changelog? @alessio , @clevinson ?

Reply:
Ping.
I like @amaurym  statement, so if no objection I will remove this changelog entry, ok?

Reply:
OK,  so I removed this entry.